### PR TITLE
🍨 Add catering menus to the site.

### PR DESCRIPTION
**Pull request by tim-schilling**
Screenshot 2024-09-15 at 19-39-44 Catering Menus

### CODE REVIEW

**Comment by tim-schilling** [comment]
Ack, I forgot to link the schedule to these.

**Comment by sodevious** [comment]
A+ on the screenshots 😝

**Review by sodevious — APPROVED** [review]

File: src/catering-menus.html [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Danishes
+- Muffins
+- Croissants
+- Coffee
+- Orange Juice
+- Water
+{% endmarkdown %}
```
Comment (drewbrew) [inline src/catering-menus.html:39]: What, you didn't want to make HTML lists? 😂

File: src/catering-menus.html [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+          <h2 id="monday-afternoon-break" class="mb-4 text-xl font-bold font-heading lg:text-3xl">
+            Monday - Afternoon Break
+          </h2>
+        </hgroup>
+
+        <div class="prose lg:prose-lg">
+{% markdown %}
+- Assorted Ice Cream Novelties
+- Peanuts
+- Crackerjacks
```
Comment (drewbrew) [inline src/catering-menus.html:162]: Isn't cracker jack two words?

**Review by drewbrew — APPROVED** [review]

File: src/catering-menus.html (hunk 2 above)
Comment (tim-schilling) [reply src/catering-menus.html:162]: I copy pasted from the contract, then reordered for the highlights to be first. I'm good with any changes you'd like to suggest.

**Review by PeterGrand — APPROVED** [review]

**Pull request merged**